### PR TITLE
server authentication profile updates

### PR DESCRIPTION
**Pull request by lachellel**
initial branch for #80

serial number update

stay tuned - each change will be separate commit

### CODE REVIEW

File: certificate-profile-server-authentication.md [hunk 1]
```diff
@@ -15,8 +15,8 @@
 | **Extension** |  **Required**   | **Critical** | **Value**                             |
 | Authority Key Identifier  | Mandatory | False |  Octet String: Derived using the SHA-1 hash of the Issuer’s public key in accordance with RFC 5280.  Must match SKI of issuing CA Certificate|
 | basicConstraints   | Mandatory | True | cA=False (Reference Section 7.1.2.3 sub-part d)|
-| Subject Key Identifier   | Mandatory | False |  Octet String <br> Derived using SHA-1 hash of the public key  |
-| Key Usage   | Mandatory | True | **Required Key Usage:** <br> Digital Signature <br><br> **Optional Key Usage:** <br> Key Encipherment for RSA Keys <br> Key Agreement for Elliptic Curve <br><br>**Prohibited Key Usage:** <br> keyCertSign and cRLSign |
+| Subject Key Identifier   | Mandatory | False |  Octet String <br> Derived using SHA-1 hash of the public key in accordance with RFC 5280 |
+| Key Usage   | Mandatory | False | **Required Key Usage:** <br> Digital Signature <br><br> **Optional Key Usage:** <br> Key Encipherment for RSA Keys <br> Key Agreement for Elliptic Curve <br><br>**Prohibited Key Usage:** <br> keyCertSign and cRLSign |
```
Comment (lachellel) [inline certificate-profile-server-authentication.md]: My mistake

The BRD does NOT specify Key Usage shall be marked non-critical
The updates made in commit e13e79cc8995a1182ecac3e2daffdf3cc637148a made this non-critical

- Which?  
- More importantly - what is the purpose and driver?

**Comment by konklone** [comment]
@lachellel Ready for review and merge?

**Comment by lachellel** [comment]
merge for review
will need to update more

**Comment by lachellel** [comment]
@konklone merge? (sep of duties :+1: )

**Comment by konklone** [comment]
:+1:

**Pull request merged**